spec/services/claim/calculate_amount_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+describe Claim::CalculateAmount do
+  let!(:claim) { create(:claim, school_id: school.id, reference: "12345678") }
+
+  let!(:region) { create(:region, name: "Inner London", claims_funding_available_per_hour: Money.from_amount(53.60, "GBP")) }
+  let!(:school) { create(:claims_school, :claims, name: "School name 1", region_id: region.id, urn: "1234") }
+
+  describe "#call" do
+    it "calculates the claim amount" do
+      create(:mentor_training, claim_id: claim.id, hours_completed: 10)
```

Comment:
You can just pass claim here as the factory should know about the association.
```suggestion
      create(:mentor_training, claim:, hours_completed: 10)
```